wgpu-types/src/lib.rs

```diff
@@ -2811,3 +2811,33 @@ bitflags::bitflags! {
         const COMPUTE_SHADER_INVOCATIONS = 0x10;
     }
 }
+
+/// Argument buffer layout for draw_indirect commands.
+#[repr(C)]
+#[derive(Clone, Debug)]
+pub struct DrawIndirectArgs {
+    vertex_count: u32,
```

Comment:
I wonder if these should be public? I.e. imagine someone using rust-gpu with wgpu-rs, they'd just `use` this structure from their shader modules and fill it up.

Reply:
dunno, had them private initially and in render.rs/compute.rs respectively. @cwfitzgerald suggested to have them in types which I took as making them public. Actually, I think it can be useful for them to be public for e.g. prefilling on host side a `Buffer` that is then adjusted in a shader. Also - more trivially - the same usecase we have here, to have something to use `mem::size_of` on

Reply:
Sorry, you misunderstood. I mean, the fields should be public as well, not just the struct types themselves.

Reply:
ah ofc, my fault. Fixed and took over the docs for the fields from WebGPU spec.

Also briefly looked at the doc in wgpu-rs. Slight differences there in field naming and one mistake there:
https://github.com/gfx-rs/wgpu-rs/blob/master/src/util/encoder.rs#L77
`DrawIndexedIndirect` is described there to have `vertex_count` instead of an `index_count` (also quickly checked with DX12 and Vulkan docs; yes it's an **index** count)
We can probably update that doc to point to these arg types now anyways :)